Documentation/MixedRealityConfigurationGuide.md

```diff
@@ -132,23 +132,19 @@ The Mixed Reality Project provides a full featured Teleportation system for mana
 ## Spatial awareness settings
 
 The Mixed Reality Project provides a rebuilt spatial awareness system for working with spatial scanning systems in the project which is selected by default.
-You can view the architecture behind the [MRTK Spatial awareness system here](../Documentation/Architecture/SpatialAwareness/SpatialAwarenessSystemArchitecture.md).
-
-> The MRTK also allows you to write your own spatial awareness system and you can use the selection below to switch the system used without rewriting the toolkit. For more information on writing your own systems, [please see this guide to building registered services]() (Coming soon)
+You can view the architecture behind the [MRTK Spatial awareness system here](../Documentation/Architecture/SpatialAwareness.md).
 
 <img src="../Documentation/Images/MixedRealityToolkitConfigurationProfileScreens/MRTK_SpatialAwarenessSystemSelection.png" width="650px" style="display:block;">
 
-The Mixed Reality Toolkit spatial awareness configuration lets you tailor how the system starts, whether it is automatically when the application starts or later programmatically as well as setting the extents for the field of view.
+The Mixed Reality Toolkit spatial awareness configuration lets you tailor how the system starts, whether it is automatically when the application starts
```

Comment:
It is the observers that have the configuration settings re: start behavior, etc. The system only manages observers.

Reply:
I don't think this section goes into any degree of specificity about observers and stuff - this is purely saying "hey look at this profile section in the inspector to configure spatial awareness"